marshmallow_recipe/bake.py

```diff
@@ -47,105 +48,112 @@ class _SchemaTypeKey:
 _schema_types: dict[_SchemaTypeKey, type[m.Schema]] = {}
 
 
+class _FieldDescription(NamedTuple):
+    field: dataclasses.Field
+    subscripted_type: TypeLike
+    metadata: Metadata
+
+
 def bake_schema(
-    cls: type,
+    t: type,
```

Comment:
I suggest not renaming cls to t here. Why should we?

Reply:
done